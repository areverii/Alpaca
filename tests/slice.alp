function void main(<string> args):
    <string> x
    <string> y
    x = ["a", "b", "c", "f"]
    y = slice(x, 1, 3)
    # should return ["b", "c"]
    printi(len(y))
    prints(y[0])
    prints(y[1])

    y = slice(x, 0, 4)
    # should return ["a", "b", "c", "f"]
    printi(len(y))
    prints(y[0])
    prints(y[1])
    prints(y[2])
    prints(y[3])
    
end

