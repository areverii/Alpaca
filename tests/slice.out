2
b
c
4
a
b
c
f
